nservicebus/upgrades/5to6/distributor.md

```diff
@@ -10,8 +10,7 @@ redirects:
  - nservicebus/scalability-and-ha/distributor/upgrading-the-distributor
 ---
 
-The [distributor](/nservicebus/msmq/distributor) has been deprecated and replaced by [sender-side distribution](/nservicebus/msmq/sender-side-distribution.md) with NServiceBus Version 6. Before upgrading, consider the current limitations of sender-side distribution mode.
-
+The [distributor](/nservicebus/msmq/distributor) has scaling limitations. For better scalability the distributor should be replaced by [sender-side distribution](/nservicebus/msmq/sender-side-distribution.md) with NServiceBus Version 6. Before upgrading, consider the current limitations of sender-side distribution mode.
```

Comment:
How about the below:

The [distributor](/nservicebus/msmq/distributor) has known performance limitations based on its design. To achieve better throughput for scalability requirements, the [sender-side distribution feature](/nservicebus/msmq/sender-side-distribution.md) available with NServiceBus Versions 6 and above can be used as an alternative approach. However, before switching out to use one feature over the other, carefully consider the limitations of both.